styles/styles.css

```diff
@@ -56,32 +43,100 @@
 }
 
  :root {
-  /* colors */
-  --c-white: #fff;
-  --link-color: #004FBC;
-  --link-hover-color: #0b326f;
-  --background-color: var(--c-white);
-  --background-hover-color: #E1DFDD;
-  --background-light-gray: #ebebeb;
-  --overlay-background-color: #eee;
-  --highlight-background-color: #ccc;
-  --volvo-text-gray: #53565A;
-  --volvo-text-black: #000;
-  --volvo-text-light-gray: #A7A8A9;
-  --text-color: var(--volvo-text-gray);
-  --btn-background-hover: #e1dfdd;
-  --btn-border-hover: #030333;
-  --input-background-color: #f1f0ef;
-
   /* fonts */
   --ff-volvo-novum-medium: 'VolvoNovum-Medium', 'VolvoNovum-Medium Fallback', sans-serif;
   --ff-volvo-novum: 'VolvoNovum', 'VolvoNovum Fallback', sans-serif;
   --ff-volvo-broadprodigital: 'VolvoBroadProDigital', 'VolvoBroadProDigital Fallback', sans-serif;
   --ff-fontawesome: 'fontawesome', 'fontawesome Fallback';
-  --body-font-family: var(--ff-volvo-novum);
-  --heading-font-family: var(--ff-volvo-novum-medium);
-  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+  --font-family-body: var(--ff-volvo-novum);
+  --font-family-heading: var(--ff-volvo-novum-medium);
+  --font-family-fixed: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+
+  /* REDESIGN */
+
+  /* Volvo Broad Pro Digital, 60px, Uppercase, 3.6px,#53565A */
+  --f-large-title-font-size: 3.75rem;
```

Comment:
We are missing `line-height` or it is just `1.0` (`100%`) for every heading?

Reply:
In the document they [provided](https://projects.netcentric.biz/wiki/download/attachments/441749354/Font%20and%20Color%20Mapping-v15-20210630_164056%5B43%5D.docx?version=1&modificationDate=1690273815593&api=v2), which can be also found here: [https://projects.netcentric.biz/wiki/display/VOLVOFN/Infrastructure](https://projects.netcentric.biz/wiki/display/VOLVOFN/Infrastructure) the line-height is not provided.